weed/filer2/cassandra/cassandra_store.go

```diff
@@ -126,6 +125,10 @@ func (store *CassandraStore) DeleteFolderChildren(ctx context.Context, fullpath
 	return nil
 }
 
+func (store *CassandraStore) ListDirectoryPrefixedEntries(ctx context.Context, fullpath util.FullPath, startFileName string, inclusive bool, limit int, prefix string) (entries []*filer2.Entry, err error) {
+	return nil, fmt.Errorf("UNSUPPORTED")
```

Comment:
create a constant error for the unsupported

Reply:
already

Reply:
```
const UnsupportedError = fmt.Errorf("UNSUPPORTED")
```

Reply:
> ```
> const UnsupportedError = fmt.Errorf("UNSUPPORTED")
> ```

I try
```
weed/filer2/filer.go:23:2: const initializer fmt.Errorf("UNSUPPORTED") is not a constant
```

Reply:
using var is fine

Reply:
ready

Reply:
see https://golang.org/pkg/io/#pkg-variables